troubleshooting.rst

```diff
@@ -41,6 +42,28 @@ File picking doesn't work or crashes
 You need to install the :code:`zenity` utility:
 - Ubuntu - :code:`sudo apt-get install zenity`
 
+Game starts with Qt UI, but not with regular launcher
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The Qt UI stores the game it's own directory per version in :code:`~/.local/share/mcpelauncher/versions.
+
+:code:`mcpelauncher-client` expects :code:`libs` and :code:`assets` directories in :code:`~/.local/share/mpelauncher` per default.
+
+You have two options:
+ - Copy, move or simlink those directories of your desired version to that location.
```

Comment:
I don't recommend doing that, so perhaps remove this suggestion (leave only the other option)